src/pages/ProductsPage/FiltersSection.js

```diff
@@ -0,0 +1,174 @@
+const FiltersSection = ({ categories, filtersState, filtersDispatch }) => {
+  const { sortBy, price, categories: categoriesState, rating } = filtersState;
+  return (
+    <aside className="filters">
+      <div className="sticky-container">
+        <div className="aside-header text-md">
+          <span>Filters</span>
+          <button
+            className="btn btn-transparent btn-sm"
+            onClick={() => filtersDispatch({ type: "CLEAR_FILTERS" })}
+          >
+            Clear all
+          </button>
+        </div>
+
+        <hr className="divider" />
+
+        <strong className="aside-subheader text-sm">Sort By</strong>
+        <div className="filter-container">
+          <input
+            type="radio"
+            id="popularity"
+            name="sort"
+            checked={sortBy === "POPULARITY"}
+            onChange={() =>
+              filtersDispatch({ type: "SORT_BY", payload: "POPULARITY" })
+            }
+          />
+          <label htmlFor="popularity">Popularity</label>
+        </div>
+        <div className="filter-container">
+          <input
+            type="radio"
+            id="low-to-high"
+            name="sort"
+            checked={sortBy === "LOW_TO_HIGH"}
+            onChange={() =>
+              filtersDispatch({ type: "SORT_BY", payload: "LOW_TO_HIGH" })
+            }
+          />
+          <label htmlFor="low-to-high">Price - Low to High</label>
+        </div>
+        <div className="filter-container">
+          <input
+            type="radio"
+            id="high-to-low"
+            name="sort"
+            checked={sortBy === "HIGH_TO_LOW"}
+            onChange={() =>
+              filtersDispatch({ type: "SORT_BY", payload: "HIGH_TO_LOW" })
+            }
+          />
+          <label htmlFor="high-to-low">Price - High to Low</label>
+        </div>
+
+        <hr className="divider" />
+
+        <label htmlFor="price" className="aside-subheader text-sm">
+          <strong>Price</strong>
+        </label>
+        <input
+          type="range"
+          className="slider"
+          min="0"
+          max="500"
```

Comment:
we can also dynamically get the max price from the products array using `map` & `Math.max()`